packages/manager/modules/billing/src/payment/method/add/views/legacyBillingAddress/translations/Messages_fr_FR.json

```diff
@@ -4,5 +4,8 @@
   "billing_payment_method_add_bank_account_address_zip": "Code postal",
   "billing_payment_method_add_bank_account_address_city": "Ville",
   "billing_payment_method_add_bank_account_owner_name": "Nom et prénom du propriétaire ou nom de l'organisation propriétaire",
-  "billing_payment_method_add_bank_account_owner_address": "Adresse du propriétaire du compte"
+  "billing_payment_method_add_bank_account_owner_address": "Adresse du propriétaire du compte",
+  "billing_payment_method_add_bank_account_owner_notification_info": "Vous allez recevoir dans les 24 à 48h sur l’adresse mail de votre compte OVHcloud, un e-mail comportant un lien de signature électronique de votre mandat de prélèvement.",
+  "billing_payment_method_add_bank_account_owner_info": "Attention, le propriétaire du compte bancaire doit être identique au propriétaire du compte OVHcloud.",
+  "billing_payment_method_add_bank_account_owner_confirmation_info": "Une fois réceptionné par nos services, votre mandat sera approuvé. Vous pourrez alors utiliser ce moyen de paiement validé."
```

Comment:
I am not sur is this sentence is true.
For me once the customer sent bank details to OVHcloud team, the team can agree/disagree the sent informations.
I suggest this: